frontend/src/shared/components/tasque-team-select/models.ts

```diff
@@ -0,0 +1,6 @@
+﻿export interface UserCircle {
+  email: string,
+  username: string,
+  profileURL: string,
+  avatarURL: string | null,
```

Comment:
avatarURL should not be null (here and at other places in the code). We will assign something by default (e.g. the first letter of the name)

Reply:
Made avatarURL not nullable